### DISCUSSION TITLE
Rowspan on rows

### DISCUSSION THREAD

**Discussion by amhassen97** [KevinVandy/material-react-table#147] [Q&A] [answered]
Hi

Love the library, very intuitive to use. I was wondering if there is way to add rowspans to rows, something like the grouping functionality, but instead of collapsing the data, the grouped column data gets displayed using a rowspan?

Something like this:

![image](https://user-images.githubusercontent.com/50408869/194757200-37d533a9-1990-4674-89e1-77915dcbcbf7.png)

There is a React/Tansack-table v7 codepen here which accomplishes the example above, but as someone who is new to Tanstack table in general, I can't seem to translate the functionality between v7 and v8. The codepen can be found here:

https://codesandbox.io/s/flattening-row-with-react-table-prkqb?file=/src/App.tsx

Any help is greatly appreciated!

**Comment by KevinVandy** [comment, marked as answer]
This library wasn't really built with rowSpan capabilities in mind, and neither was TanStack Table v8. Maybe it is something for me to consider for a future feature when pagination is also not enabled.

**Comment by amhassen97** [comment]
Ah I see thanks, really appreciate it! Would be cool if you could integrate it in a future version.
